spec/system/search/display_articles_search_spec.rb

```diff
@@ -28,7 +28,6 @@
     expect(page).to have_selector("button[data-reactable-id=\"#{found_article_one.id}\"]")
     expect(page).to have_content("5 min read")
     expect(find_link("#ruby")["href"]).to include("/t/ruby")
-    expect(find_link(found_article_one.user.name)["href"]).to include(found_article_one.username)
```

Comment:
I've removed this because the broader tests required for making sure the right preview details are added are now in Cypress